code/style.css

```diff
@@ -1,4 +1,293 @@
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  margin-bottom: 1rem;
+}
+
+.logo {
+    margin: 0 2em;
+}
+
+.navigation  {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.navigation a {
+  display: block;
+  padding: 1em;
+  color: white;
+  text-decoration: none;
+  letter-spacing: .45rem;
+}
+
+.navigation a:hover {
+  background: #00ace6;
+}
+
+/* BIG NEWS SECTION */ 
+
+.latest-news-title {
+  font-size: 40px;
+  padding: 0 0 2rem 0;
+}
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  width: auto;
+  height: auto;
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
+  width: 1200px;
+  height: 1000px;
+
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding: 2rem;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  width: 1000px;
+  padding-top: 1rem;
+}
+
+.big-news-image {
+  width: 1000px;
+ }
+
+.big-news-title {
+  font-size: 32px;
+  padding-bottom: 1em;
+}
+
+.big-news-text {
+  font-size: 14px;
+  display: block;
+}
+
+.big-news-text a {
+  font-size:20px;
+  font-family:'Bungee'; 
+  text-decoration: underline;
+}
+
+.big-news-text a:hover {
+  color:#00ace6;
+}
+
+.little-title {
+  padding-top: 2rem;
+  font-size: 12px;
+  color:rgba(126, 126, 126, 0.61);
+}
+
+
+/* OTHER NEWS SECTIONS*/ 
+
+.other-news {
+  display: flex;
+  width: 1200px;
+  flex-wrap: wrap;
+  padding:1rem;
+}
+
+.news-card {
+  width: 380px;
+  height: 400px;
+
+
+}
+
+.other-news-image {
+  width: 340px;
+}
+
+.other-news-title {
+  padding: 1rem 0;
+}
+
+.other-news-text {
+  font-size: 12px;
+  width: 340px;
+}
+
+.other-news a {
+  font-size: 14px;
+  font-family: 'Bungee';
+  text-decoration: underline;
+  display: block;
+  padding-top:1rem;
+}
+
+.other-news a:hover {
+  color:#00ace6;
+}
+
+/* FOOTER */
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 8px 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding:1rem;
+}
+
+footer p {
+  font-family: 'Bungee';
+  font-size: 16px;
+  letter-spacing: .10rem;
+  padding:1em;
+}
+
+.project {
+  font-family: 'Roboto';
+  text-align: center;
+  font-size: 10px;
+}
+
+footer a:hover {
+  color:#00ace6;
+}
+
+
+
+
+
+
+/* ******MEDIA QUERIES****** */ 
+
+
+
+
+
+
+/* STYLE FOR TABLET */
+
+@media (max-width: 1259px) {
+  .flex-container {
+    flex-direction: column;
+  }
+  
+  .logo {
+    margin: 0;
+  }
+  
+  .navigation  {
+    width: 100%;
+    justify-content: space-around;
+  }
+  
+  .news {
+    width: auto;
+    height: auto;
+  }
+
+  .big-news-text {
+    font-size: 14px;
+  }
+
+  .big-news-text a {
+    font-size: 18px;
+  }
+
+
+
+}
+
+
+/* STYLE FOR MOBILE */
+
+@media (max-width: 1000px) {
+ 
+  .logo {
+    margin: .25em 0;
+  }
+
+  .navigation {
```

Comment:
**Mobile**

Navbar is taking up almost 60% of the screen in mobile mode. Make logo little bit lower so you see more of the latest news?   

**Potential solution:** Decrease size on logo. Height: 100px;

Reply:
Yes! It looks much better! Thanks 💯